client/src/lib/api/viame.service.ts

```diff
@@ -78,7 +88,7 @@ function runImageConversion(folder: string) {
 }
 
 function getPipelineList() {
-  return girderRest.get('viame/pipelines');
+  return girderRest.get<Record<string, Category>>('viame/pipelines');
```

Comment:
@BryonLewis I discovered this syntax recently and it really seems the most correct to me.